Presentation.Web/Tests/HomePage/login.e2e.spec.ts

```diff
@@ -0,0 +1,36 @@
+﻿import login = require("../Helpers/LoginHelper");
+import homePage = require("../PageObjects/HomePage/HomePage.po")
+
+var pageObject = new homePage();
+var navigationDropdown = pageObject.navigationDropdownMenu();
+var loginHelper = new login();
+
+describe("Can login succesfully", () => { 
+
+    beforeEach(() => {
+        pageObject.getPage();
+    });
+
+    afterEach(() => {
+        browser.driver.manage().deleteAllCookies();
+    })
+
+    it("As global admin", () => {
+        loginHelper.loginAsGlobalAdmin();
```

Comment:
Der skal vel også være en "Global admin"

Reply:
Og her er det da global admin?

Reply:
@JacobMalling Ja men du tester ikke for at menupunktet "Global admin" er i menuen :-) Samme problem med lokal admin. Når du har en af de roller skal du både have din egen profil samt mulighed for at skifte til global/lokal admin tilstand